dotCMS/src/main/java/com/dotmarketing/portlets/workflows/business/WorkflowAPI.java

```diff
@@ -241,10 +241,31 @@ public void reorderAction(final WorkflowAction action,
 
 
 
+	/**
+	 * Finds an action by Id and checking the user permissions over the workflow portlet.
+	 * The action will be validated against the user permissions.
+	 * @param id String action id
+	 * @param user     User   the user that makes the request
+	 * @return WorkflowAction
+	 * @throws DotDataException
+	 * @throws DotSecurityException
+	 */
 	public WorkflowAction findAction(String id, User user) throws DotDataException, DotSecurityException;
 
 	/**
-	 * Finds an action associated to the steps.
+	 * Finds an action by Id and checking the user permissions over the permissionable.
+	 * The action will be validated against the user permissions.
+	 * @param id String action id
+	 * @param permissionable   Permissionable Content/Content Type against who is going to be validated the permissions
+	 * @param user     User   the user that makes the request
+	 * @return WorkflowAction
+	 * @throws DotDataException
+	 * @throws DotSecurityException
+	 */
+	public WorkflowAction findActionRespectingPermissions(String id, Permissionable permissionable, User user) throws DotDataException, DotSecurityException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 154 characters long line (which is greater than 150 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00103)